app/Console/Commands/ExportEntityTypes.php

```diff
@@ -0,0 +1,563 @@
+        ];
+    }
+
+    private function collectLanguagesForConcept($concept, &$thesaurusLabels): void {
+        // Collect all labels (that are in the valid languages)
+        $concept->labels->each(function($label) use (&$thesaurusLabels, $concept) {
+            if(in_array($label->language->short_name, static::$validLanguages)) {
+                $lang = $label->language->short_name;
+                $thesaurusLabels[$concept->concept_url . "-" . $lang] = $label;
+            }
+        });
+    }
+
+    private function collectEntityRelations(array $entityTypes): array {
+        $entityTypesMap = [];
+        foreach($entityTypes as $entityType) {
+            $entityTypesMap[$entityType->id] = $entityType;
+        }
+
+        $relations = [];
+        foreach($entityTypes as $entityType) {
+            $parent = $entityType;
+            $children = $entityType->sub_entity_types;
+            foreach($children as $child) {
+                // We only include relations that we are going to transfer
+                if(!isset($entityTypesMap[$child->id])) {
+                    continue;
+                }
+                $relation = EntityTypeRelation::where('parent_id', $parent->id)
+                    ->where('child_id', $child->id)
+                    ->first();
+                $relations[] = $relation;
+            }
+        }
+        return $relations;
+    }
+
+    private function transferData($user, $data): void {
+        $this->printSectionHeading("Transfering Data Models to Transfer Database");
+        $this->printHeading("Transfering Thesaurus Concepts", $data['thesaurusConcepts']);
+        $this->transferThConcepts($user, $data['thesaurusConcepts']);
+        $this->printSeparator();
+        $this->printHeading("Transfering Thesaurus Labels", $data['thesaurusLabels']);
+        $this->transferThLabels($user, $data['thesaurusLabels']);
+        $this->printSeparator();
+        $this->printHeading("Create Thesaurus Broader/Narrower Relations");
+        $this->transferBroaderNarrowerRelations($user);
+        $this->printSeparator();
+        $this->printHeading("Transfering Entity Types", $data['entityTypes']);
+        $this->transferEntityTypes($user, $data['entityTypes']);
+        $this->printSeparator();
+        $this->printHeading("Entity Type Relations");
+        $this->transferEntityTypeRelations($data['entityTypeRelations']);
+        $this->printSeparator();
+        $this->printHeading("Transfering Attributes", $data['attributes']);
+        $this->transferAttributes($user, $data['attributes']);
+        $this->printSeparator();
+        $this->printHeading("Transfering Entity Type Attributes Relation", $data['entityTypeAttributeRelations']);
+        $this->transferEntityTypeAttributeRelations($user, $data['entityTypeAttributeRelations']);
+        $this->printSeparator();
+    }
+
+    private function selectOrCreateTransferUser(): User {
```

Comment:
We shouldn't create a dummy user. This is going to be a mess if data is stored in a repository.